src/app/modules/core/manager/reservation/components/details/manager-reservation-details.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core'
```

Comment:
Please, rename the file, the component class, all the related files and the selector in `reservation-details` instead of `manager-reservation-details`. Most probably this component will be shared with the landlord module. Thanks.